### DISCUSSION TITLE
Question about CoC autocomplete dialog

### DISCUSSION THREAD

**Discussion by Warkanlock** [neoclide/coc.nvim#3638] [Q&A] [answered]
I was trying to get used to the idea of CoC and I was wondering,

**What does this mean in terms of CoC semantics?**

![image](https://user-images.githubusercontent.com/13340320/153710712-3b057aad-52d5-43d4-b418-553efa7bc2db.png)

I mean, ``[TSC]`` and the keyword on the left (``F``, ``f``, sometimes is a ``m`` or a ``v``)

I know that this could be a silly question...

Thanks in advance

**Comment by chemzqm** [comment, marked as answer]
It's tsserver, use `:CocList sources` to check available sources.

**Reply by Warkanlock** [reply]
Thanks, man, I was going crazy trying to crack those initials

**Reply by chemzqm** [reply]
https://github.com/neoclide/coc.nvim/wiki/Completion-with-sources#completion-sources
